package.json

```diff
@@ -11,10 +11,15 @@
 		"build": "tsc -p src",
 		"dev": "yarn build && node .",
 		"clean": "rimraf dist",
-		"watch": "tsc -b src -w",
+		"watch": "tsc-watch -b src --onSuccess \"node .\"",
+		"predocker:watch": "yarn",
+		"docker:watch": "yarn watch",
 		"lint": "eslint src --ext ts --fix",
 		"test": "jest",
-		"typeorm": "node -r module-alias/register ./node_modules/typeorm/cli.js"
+		"typeorm": "node -r module-alias/register ./node_modules/typeorm/cli.js",
+		"predev:typeorm": "yarn build",
+		"dev:typeorm": "yarn typeorm",
```

Comment:
This is to assure you don't need to build yourself before running migrations, you can just do `yarn run dev:typeorm migrations:run` and it will do a build using the `pre` hook